app/views/unattended/provisioning_templates/PXEGrub/jumpstart_default_pxegrub.erb

```diff
@@ -4,7 +4,10 @@ name: Jumpstart default PXEGrub
 model: ProvisioningTemplate
 oses:
 - Solaris
-%>
+description: |
+  The template to render PXEGrub bootloader configuration for Solaris.
```

Comment:
Nitpick: The software is named Grub (version 1 or 2). Historically, Foreman had template kind named "PXELinux" so we continued with this with "PXEGrub" and "PXEGrub2". But technically it is incorrect, nothing is named like this. We could use the correct term in all descriptions, bonus: it is shorter.